### ISSUE TITLE
api.resetRowHeights() doesn't work because of cellRenderer in 21.0.1

### ISSUE THREAD

**Issue by sally4388you** [ag-grid/ag-grid#3217]
Thanks in advance! Appreciate any help you can provide!

**I'm submitting a ...**
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```
**Providing a Reproducible Scenario**
- Plunker: http://plnkr.co/edit/O5RtSamMEkQHKAzMWAcJ?p=preview

**Description**
- Already set `white-space: normal !important;`
- Already set `autoHeight: true`
- Using cellRenderer
- ag-Grid version: 21.0.1

It's a really simple scenario. It **was able to** reset row height for columns **with cellRenderer**, but after the recent update, it is not able to do it anymore. I really looked into it and found out that it is because `createCellRendererInstance` function doesn't work for components with cellRenderer. Though I don't actually understand what is going on...

Quote from ag-grid-community/src/ts/rendering/cellComp.ts
```
809  private createCellRendererInstance(useTaskService = false): void {
```
The cell will be rendered again if `useTaskService` is false. But when the function gets called, `useTaskService` is always true.

**Current behavior**
From plunker, the first row doesn't reset row height.

**Expected behavior**
The first row should be taller than other rows because the data for `model` column is longer than 200px.

**Comment by davis4acca** [comment]
Hi,

Thank you for your email. We are aware of this issue and have a feature request in our backlog:

You can follow the status of feature requests, bugs and releases using our pipeline: 
https://www.ag-grid.com/ag-grid-pipeline/

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.

Note that this is parked as of this time, just because of its complexity, which means that even though we are aware of this issue, we don't have any easy solution as of this time

This is the reference and summary of the relevant issue:


AG-3068	cellRenderer + autoHeight not working in the latest version

**Comment by amashar** [comment]
@davis4acca  : is this issue now resolved? I am facing similar thing in ag-grid 23.2.1

**Comment by destradafilm** [comment]
I still see this and I'm on `ag-grid-community: 27.2.0`
